src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import React from 'react';
+
+interface TabsType {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type Props = {
+  tabs: TabsType[],
+  selectedTabsId: string,
+  onTabSelected: React.MouseEventHandler<HTMLAnchorElement>,
+  selectedTab: TabsType | undefined,
+};
+
+export const Tabs: React.FC<Props> = React.memo(({
+  tabs,
+  selectedTabsId,
+  onTabSelected,
+  selectedTab,
+}) => (
+  <>
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(({ id, title }) => (
+          <li
+            key={id}
+            className={id === selectedTabsId ? 'is-active' : ''}
+            data-cy="Tab"
+          >
+            <a
+              href={`#${id}`}
+              data-cy="TabLink"
+              onClick={onTabSelected}
+            >
+              {title}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </div>
+
+    <div className="block" data-cy="TabContent">
+      {selectedTab?.content}
```

Comment:
Instead of using `selectedTab?.content`, you can find the selected tab's content using `selectedTabsId`. For example, you can use `tabs.find(tab => tab.id === selectedTabsId)?.content`.

Reply:
@etojeDenys , I have tabs.find() operation in the App and I didn't want to dublicate this in the Tabs.